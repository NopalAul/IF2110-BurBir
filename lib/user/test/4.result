lib/user/test/4.in lib/user/test/4.out: TRUE
