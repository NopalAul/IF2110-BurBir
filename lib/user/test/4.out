2
asep 1
gacorz
23/11/2023 21:52:75
deni 2
nigger hitam
23/11/2023 21:52:75
ini draft, semoga gacor kang
23/11/2023 21:52:75
